axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceTermServiceImpl.java

```diff
@@ -95,7 +95,7 @@ public class InvoiceTermServiceImpl implements InvoiceTermService {
   protected InvoicePaymentCreateService invoicePaymentCreateService;
   protected UserRepository userRepo;
   protected ScaleServiceAccount scaleServiceAccount;
-  private static final int ALTERNATIVE_SCALE = 10;
+  private static final int COMPUTATION_SCALING = AppBaseService.COMPUTATION_SCALING;
```

Comment:
You don't have to define it here, you can juste use `AppBaseService.COMPUTATION_SCALING` everywhere

Reply:
Resolved